content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/index.md

```diff
@@ -548,7 +548,8 @@ to hold the configuration of the NGINX server:
 
     {{< warning >}}
     The secret name **should not** begin with `istio` or `prometheus`, and
-    the secret **should not** contain a `token` field.
+    the secret **should not** contain a `token` field, and
+    the secret **should not** end with `-cacert`.
```

Comment:
The secret name should not ... ?

```suggestion
    the secret name **should not** end with `-cacert`.
```

Reply:
> The secret name should not ... ?
Yes, I tested it in v1.8.0, when the secret name suffixed with '-cacert', "upstream connect error or disconnect/reset before headers. reset reason: connection failure, transport failure reason: TLS error: Secret is not supplied by SDS" will be returned if accessing the service